### ISSUE TITLE
some new bugs

### ISSUE THREAD

**Issue by scatterp2** [peatio/peatio#598]
using muhammednagy git
when trying to sell or buy added pairs the word HIGH appears and the action fails
when trying to buy or sell ETH orders are not matched by matching engine 
there is a translation error on the balance drop down for ethreum 

see server.cryex24.com if its down I am working on it feel free to contact me 
here https://gitter.im/peatio/peatio

**Comment by MaffeProg** [comment]
Yep you're right. I am also searching for a solution. Translation already fixed just need to add some eth information in the en file

**Comment by scatterp2** [comment]
MaffleProg can you be a little more specific which file which section so it can be resolved as an issue and pushed to the git

**Comment by scatterp2** [comment]
I solved the translation error ethereum should be "ether"

**Comment by muhammednagy** [comment]
can you post the errors in console?

**Comment by scatterp** [comment]
when trying to buy or sell ETH orders are not matched by matching engine
for this there are no errors simply it does not match

**Comment by scatterp** [comment]
when trying to sell or buy added pairs the word HIGH appears and the action fails
nothing in console this appears in the market window

**Comment by scatterp** [comment]
so add currencys to order.rake (this worked for me not sure about others) 
add currencys to currency.yml
bundle exec rake db:setup 
add currencys to markets.yml
and update your balance using mysql 
see my github for more..
